playbooks/README.md

```diff
@@ -0,0 +1,21 @@
+# openshift-ansible playbooks
+
+In summary:
+
+- [`byo`](byo) (_Bring Your Own_ hosts) has the most actively maintained
+  playbooks for installing, upgrading and performing others tasks on OpenShift
+  clusters.
+
+And:
+
+- [`adhoc`](adhoc) is a generic home for playbooks and tasks, unsupported and
+  not maintained.
+- [`common`](common) was meant as a generic set of playbooks that would run
+  against different "providers" when the now deprecated
+  [`bin/cluster`](../bin/cluster) tool was supported.
```

Comment:
I moved `common` up, next to `byo`, and made the summary description less verbose, then added a better explanation under `common/README.md` to mention both uses.